docs/sources/style-guide/style-conventions/index.md

```diff
@@ -130,54 +131,10 @@ A tip describes a more efficient or alternate way of doing something. Tips are r
 
 A caution warns the user to proceed with caution. A caution emphasizes a course of action's potential downsides.
 
-The markdown syntax for a caution is `> **Caution**` and will appear in the published output as shown in the following example:
-
 > **Caution:** By disabling authentication requirements, anyone can access your Grafana instance. There is a considerable security risk associated with this.
 
 ### Warnings
 
-A warning informs the user not to do something.  Warnings are usually reserved for actions that, if performed, could cause harm to hardware, software, or data.  
```

Comment:
Why remove this?

Reply:
+1.  My last job had way too many warnings in the documentation because engineers did not understand that there is a standard for when to use them.

Reply:
Fixed in latest commit